FreshRSS/FreshRSS
# Improved: template.css (colors)

Similar to #4636.

All colors are now collected within CSS variables in `template.css`
It gives a better overview of used colors and adjusting in.
All variables starts with `--template` to mark them as template.css variables. They could be used by the themes to easily adjust them instead of copy&paste each CSS line.

Changes proposed in this pull request:

- CSS
- little improvements for the default configs.

How to test the feature manually:

1. check the other themes
2. hopefully nothing / not much changed there


Pull request checklist:
